test/Libraries/Microsoft.Extensions.AI.Evaluation.NLP.Tests/BLEUEvaluatorTests.cs

```diff
@@ -0,0 +1,113 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Microsoft.Extensions.AI.Evaluation.NLP;
+using Xunit;
+
+namespace Microsoft.Extensions.AI.Evaluation.NLP.Tests;
+
+#pragma warning disable AIEVAL001 // Type is for evaluation purposes only and is subject to change or removal in future updates. Suppress this diagnostic to proceed.
+
+public class BLEUEvaluatorTests
+{
+    [Fact]
+    public async Task EvaluateAsync_ReturnsPerfectScoreForIdenticalText()
```

Comment:
I'd prefer to leave out the `EvaluateAsync` prefix so that the test names are shorted in the Test Explorer :)